packages/firestore/lib/modular/index.d.ts

```diff
@@ -10,6 +10,7 @@ import Query = FirebaseFirestoreTypes.Query;
 import FieldValue = FirebaseFirestoreTypes.FieldValue;
 import FieldPath = FirebaseFirestoreTypes.FieldPath;
 import PersistentCacheIndexManager = FirebaseFirestoreTypes.PersistentCacheIndexManager;
+import AggregateQuerySnapshot = FirebaseFirestoreTypes.AggregateQuerySnapshot;
```

Comment:
I was curious where this was defined as [the types looked pretty intense](https://github.com/firebase/firebase-js-sdk/blob/e3e20784503347ba5cab1ead3a7b73df87125cb4/packages/firestore/src/lite-api/aggregate_types.ts#L68-L119) - looks like we get this for free from upstream? (if so: fantastic)

Reply:
Ah - I see they were split to the other (non-modular) index.d.ts since count was implemented before modular was a thing here - types look better now in that file +1